Cloud changelog Feb 24th release (#4316)
mattermost/docs: source/cloud/cloud-administration/cloud-changelog.md

This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.

## Release 2021-02-24

### Highlights

#### Custom Statuses

>  - Users can set a custom status by using the custom status dropdown in the status menu.

Reviewer: Custom Statuses allows users to add a descriptive status message and emoji that’s visible to everyone. Users now gain the flexibility to express their current status in any way they prefer. Mobile support is coming soon.

#### Support Packet Generation
 - Allows a System Admin to download a support packet which provides helpful information to our internal support team.

### Improvements

#### User Interface (UI)
 - Removed the 5-page limit on previewing PDFs.
 - Added "files" as a reserved team name.

>  - Searching for a channel url_name now shows the channel.

Reviewer 2: ```suggestion
 - Searching for a channel by URL now returns the channel.
```

>  - Users are now provided with feedback when creating a custom category name for a channel that exceeds the character limit.

Reviewer 2: ```suggestion
 - Users are now provided with feedback when creating a custom category name that exceeds the character limit.
```

#### Administration
 - Added support for compressed export files with attachments.
 - Server crashes due to runtime panics are now captured as a log line.

>  - Optimized Direct Message creation by fetching all users involved in a single Database call.

Reviewer 2: ```suggestion
 - Optimized Direct Message creation by fetching all users involved in a single database call.
```

>  - A change in a user's ``NotifyProps`` during the user import process will not send an email notification. This is done to make it consistent with other parts of the import process where a change in user's attributes would also not send any notifications.

Reviewer 2: ```suggestion
 - During the user import process, a change in a user's ``NotifyProps`` will not send an email notification. This is done to make it consistent with other parts of the import process where a change in user's attributes would also not send any notifications.
```

 - Implemented a job to delete unused export files.

### Bug Fixes
 - Fixed an issue where demoting a user to a guest would not take effect in an environment with read replicas.
 - Fixed an issue where creation of a bot would fail due to replica lag.

>  - Fixed an issue where ``mmctl channel move`` did not allow moving private channels.

Reviewer 2: ```suggestion
 - Fixed an issue where ``mmctl channel move`` did not allow private channels to move.
```

 - Fixed an issue where markdown tables did not wrap correctly.

>  - Fixed an issue where the search bar styling on dark themes was off on mobile web view.

Reviewer 2: What does "off" refer to in this specific context? Does it indicate disabled or something else?

>  - Fixed an issue where the **Main Menu** on webapp appeared more left-aligned than previously.

Reviewer 2: ```suggestion
 - Fixed an issue where the **Main Menu** on webapp appeared more left-aligned than previous releases.
```

 - Fixed an issue where sticky sidebar headings appeared under **More Unreads**.
 - Fixed an issue where the group channel icon was misaligned in the channel switcher.
 - Fixed an issue where line breaks were ignored when used with inline images.

>  - Fixed a panic when the Oauth discovery endpoint would not return a Cache-Control header.

Reviewer 2: ```suggestion
 - Fixed a panic when the OAuth discovery endpoint would not return a Cache-Control header.
```

### Known Issues

>  - Sometimes an "Unable to get role" error appears when changing a channel member role in **System Console > User Management > Channel**.

Reviewer 2: ```suggestion
 - Sometimes an "Unable to get role" error appears when changing a channel member role in **System Console > User Management > Channels**.
```

#### Support Packet Generation
 - Allows a System Admin to download a support packet which provides helpful information to our internal support team.

### Improvements

#### User Interface (UI)
 - Removed the 5-page limit on previewing PDFs.
 - Added "files" as a reserved team name.
 - Searching for a channel by URL now returns the channel.
 - Users are now provided with feedback when creating a custom category name that exceeds the character limit.

#### Administration
 - Added support for compressed export files with attachments.
 - Server crashes due to runtime panics are now captured as a log line.
 - Optimized Direct Message creation by fetching all users involved in a single database call.
 - During the user import process, a change in a user's ``NotifyProps`` will not send an email notification. This is done to make it consistent with other parts of the import process where a change in user's attributes would also not send any notifications.
 - Implemented a job to delete unused export files.

>  - Improved the websocket implementation by using epoll to manually read from a websocket connection. As a result, the number of goroutines is expected to go down by half. This implementation is only available on Linux and FreeBSD based distributions. If you are using NGINX as a proxy to Mattermost, please ensure to have ``proxy_http_version 1.1;`` in the block that handles the websocket path.

Reviewer 2: ```suggestion
 - Improved the websocket implementation by using epoll to manually read from a websocket connection. As a result, the number of goroutines is expected to go down by half. This implementation is only available on Linux and FreeBSD-based distributions. If you are using NGINX as a proxy to Mattermost, please ensure to have ``proxy_http_version 1.1;`` in the block that handles the websocket path.
```

### Known Issues
 - The slash command autocomplete options cover the input box on some reply threads on the right-hand side.

>  - Sometimes an "Unable to get role" error appears when changing a channel member role in **System Console > User Management > Channel**.

Reviewer 2: ```suggestion
 - Sometimes an "Unable to get role" error appears when changing a channel member role in **System Console > User Management > Channels**.
```
